# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/mediapipeline/moz.build
# Commit: 0b03dd9d20ac
# Full Hash: 0b03dd9d20ace235502714101097fe34d1864563
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-07-31 15:46:43
# Regressor Bug: 1493613
# File Overlap Count: 1
# Description:
#   Bug 1493613 - Update muted state through MediaStreamTrackSource. r=bwc,smaug
#   
#   This ensures all clones of the original track also receives the new muted state.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D37933
# ==============================================================================

diff -r cf84eeffed22 -r 0b03dd9d20ac media/webrtc/signaling/src/mediapipeline/moz.build
--- a/media/webrtc/signaling/src/mediapipeline/moz.build	Tue Jul 30 14:55:48 2019 +0000
+++ b/media/webrtc/signaling/src/mediapipeline/moz.build	Wed Jul 31 09:35:06 2019 +0000
@@ -13,6 +13,7 @@
     '/media/mtransport',
     '/media/webrtc',
     '/media/webrtc/signaling/src/media-conduit',
+    '/media/webrtc/signaling/src/peerconnection',
     '/media/webrtc/trunk',
     '/media/webrtc/trunk/webrtc',
     '/netwerk/srtp/src/crypto/include',